common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -444,6 +447,57 @@ public static void validateUpdateUser(Request userRequest) {
     }
   }
 
+  public static void externalIdsValidation(Request userRequest) {
+    if (userRequest.getRequest().containsKey(JsonKey.EXTERNAL_IDS)
```

Comment:
Is there a check somewhere else to make sure that "userRequest" object or "userRequest.getRequest()" value will not be null?